coreos-base/coreos-metadata/files/coreos-metadata.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=CoreOS Metadata Agent
+Requires=network-online.target
+After=network-online.target
```

Comment:
What do you mean about safety?

This service and it's parent target won't be enabled by default. If the user needs metadata, they'll have to add the dependency to their service.